.github/workflows/main.yml

```diff
@@ -77,11 +77,36 @@ jobs:
           filters: |
             beaver_terminal:
               - 'frontend/beaver_terminal/beaver_cli/**'
-  
+
   call_cli_workflow:
     needs: [linting, cli_changes]
     uses: ./.github/workflows/cli_workflow.yml
     with:
       cli_changed: ${{ needs.cli_changes.outputs.cli_changed == 'true' }}
     secrets:
-      PYPI_SECRET_TOKEN: ${{ secrets.PYPI_SECRET_TOKEN }}
\ No newline at end of file
+      PYPI_SECRET_TOKEN: ${{ secrets.PYPI_SECRET_TOKEN }}
+
+  beaver_api_changes:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      api_changed: ${{ steps.filter.outputs.beaver_api }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          base: master
+          filters: |
+            beaver_api:
+              - 'backend/apps/beaver_api/**'
+
+  call_beaver_api_workflow:
+    needs: [beaver_api, beaver_api_changes]
+    if: github.ref == 'refs/heads/master'
+    uses: ./.github/workflows/beaver_api_workflow.yml
+    with:
+      api_changed: ${{ needs.beaver_api_changes.outputs.api_changed == 'true' }}
+    secrets: inherit
```

Comment:
Do we have AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY secrets created for github ci/cd?